spring-aot/src/test/java/org/springframework/cloud/function/FunctionTypeProcessorTests.java

```diff
@@ -39,20 +40,35 @@
  */
 class FunctionTypeProcessorTests {
 
-	private static final Set<TypeAccess> ALL_MEMBERS = Set.of(TypeAccess.DECLARED_CONSTRUCTORS,
+	private static final Set<TypeAccess> METHODS_AND_CONSTRUCTORS = Set.of(TypeAccess.DECLARED_CONSTRUCTORS,
```

Comment:
Instead of testing here, could we just add an `@Bean` use case and modify `verify.sh` in one of the `cloud-function-*` sample?

Reply:
Let me try that